pallets/schemas/src/weights.rs

```diff
@@ -151,18 +105,59 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Proof: `Council::ProposalCount` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
 	/// Storage: `Council::Voting` (r:0 w:1)
 	/// Proof: `Council::Voting` (`max_values`: None, `max_size`: None, mode: `Measured`)
-	/// The range of component `m` is `[16, 65499]`.
-	fn propose_to_create_schema_v2_with_name(m: u32, ) -> Weight {
+	fn propose_to_create_intent() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `166`
 		//  Estimated: `4126`
-		// Minimum execution time: 20_014_000 picoseconds.
-		Weight::from_parts(5_538_381, 4126)
-			// Standard Error: 45
-			.saturating_add(Weight::from_parts(3_426, 0).saturating_mul(m.into()))
+		// Minimum execution time: 17_000_000 picoseconds.
+		Weight::from_parts(17_000_000, 4126)
 			.saturating_add(T::DbWeight::get().reads(4_u64))
 			.saturating_add(T::DbWeight::get().writes(4_u64))
 	}
+	/// Storage: `Schemas::NameToMappedEntityIds` (r:1 w:1)
+	/// Proof: `Schemas::NameToMappedEntityIds` (`max_values`: None, `max_size`: Some(95), added: 2570, mode: `MaxEncodedLen`)
+	/// Storage: `Schemas::IntentInfos` (r:10 w:0)
+	/// Proof: `Schemas::IntentInfos` (`max_values`: None, `max_size`: Some(13), added: 2488, mode: `MaxEncodedLen`)
+	/// Storage: `Schemas::CurrentIntentGroupIdentifierMaximum` (r:1 w:1)
+	/// Proof: `Schemas::CurrentIntentGroupIdentifierMaximum` (`max_values`: Some(1), `max_size`: Some(2), added: 497, mode: `MaxEncodedLen`)
+	/// Storage: `Schemas::IntentGroups` (r:0 w:1)
+	/// Proof: `Schemas::IntentGroups` (`max_values`: None, `max_size`: Some(31), added: 2506, mode: `MaxEncodedLen`)
+	/// The range of component `m` is `[0, 10]`.
+	fn create_intent_group(m: u32, ) -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `210 + m * (39 ±0)`
+		//  Estimated: `4055 + m * (2488 ±0)`
+		// Minimum execution time: 14_000_000 picoseconds.
+		Weight::from_parts(17_499_362, 4055)
+			// Standard Error: 37_852
+			.saturating_add(Weight::from_parts(2_552_879, 0).saturating_mul(m.into()))
+			.saturating_add(T::DbWeight::get().reads(2_u64))
+			.saturating_add(T::DbWeight::get().reads((1_u64).saturating_mul(m.into())))
```

Comment:
This is weird. I think these weights are generated when we were reading all intents for verification and might need to re-generate this to be accurate.